### DISCUSSION TITLE
Archive node on HDD (it's ok to fall behind)

### DISCUSSION THREAD

**Discussion by ofarukcaki** [erigontech/erigon#6369] [Q&A] [answered]
I know that SDD is recommended for running a node, which normally I run my own node on NVMe but here I have a different use case right now.

Can I use HDD to run an **archive node**, it is totally fine if my node falls behind N block or whatsoever. I just want to keep records of older blocks that I can't access with my full node.

If so, the initial sync time would be the same with SSD, or would be increased dramatically as well?

Thanks

**Comment by AskAlexSharov** [comment, marked as answer]
1. if default mode it likely will not "constantly fall behind" - but will stabilize 1K block behind chain tip 
2. can run only RPC without sync (by running only ./cmd/rpcdaemon)
3. can sync periodically (once a day) by --sync.loop.throttle
4. can disable something: --txpool.disable, --torrent.uploda.rate=1kb
5. Archive node need less IO than FullNode (delete old data is not free).
6. can mount/symlink `datadir/snapshots` to HDD and `datadir/chaindata` to SSD
7. if you will try sync from scratch on HDD - likely it will take too long time (if you don't have > 128GB RAM). sync on SSD, then copy to HDD is also possible. 
8. env variable like NO_SYNC=true maybe used, but we don't test it much. (likely it will not help much - bottleneck usually is disk read latency).

**Reply by ofarukcaki** [reply]
Thank you very much 🙏🏻. That will be very helpful.
